src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3344,6 +3347,21 @@
     <value>(multiple values)</value>
     <comment>Text indicating that multiple selected files have different metadata values.</comment>
   </data>
+  <data name="GitCommitAuthor" xml:space="preserve">
+    <value>Author</value>
+  </data>
+  <data name="GitLastCommitDate" xml:space="preserve">
```

Comment:
```suggestion
  <data name="LastCommit" xml:space="preserve">
```

Reply:
I think Date Committed is better to be consistent with Date modified, Date created.